app/grandchallenge/archives/urls.py

```diff
@@ -10,7 +10,7 @@
     ArchiveItemInterfaceCreate,
     ArchiveItemsList,
     ArchiveItemsToReaderStudyUpdate,
-    ArchiveItemUpdate,
+    ArchiveItemUpdateView,
```

Comment:
`ArchiveItemUpdateView` is a CBV, and has a method `.as_view` that produces a view. So, this should be kept as `ArchiveItemUpdate`.

Reply:
I am confused. The default `View` Django class has the `as_view`. Not calling this a View seems at odds with the Django framework. Moreover, it directly inherits from the Django's `FormView`.

Reply:
Not sure if this rename is still needed?

Reply:
I prefer not to include it for the reasons given before. Most of the other classes imported from this module do not have it, I would rather we stay consistent with that.